Updated instructions based on Google Docs Changes

thejimbirch opened the pull request:
I had a slightly different experience, most likely to Google changing their flow.  I thought I would update your instructions as a thank you.  Thank you!  Great tool.

Commit:
Updated instructions based on Google Docs Changes

Commit:
Remove small class from UL.  Changed <b> to <strong>

thejimbirch commented:
Added a second commit which removed class="small" from ul tags, and switched out b tags with strong tags.

Commit:
Changes List items to check if it is undefined before asking for its type - @stanleyshilov

thejimbirch commented:
Added a commit that updated the List item parsing.  Discussion in the comments at http://jimbir.ch/blog/clean-google-docs-html-programmatically

WiliTest commented:
@oazabir  please apply these changes (the script is useless without them)

djwesterfield reviewed (commented):
I'm getting this error
`TypeError: Cannot call method "getBody" of null. (line 5, file "GoogleDoc2Html")`

freejoe76 committed:
Accommodate https links

Add logic that lets the script parse and return links that start with "https://" in addition to links that start with "http://"

thejimbirch committed:
Merge pull request #2 from freejoe76/patch-1

Accommodate https links

thejimbirch committed:
Update README.md
